hedera-node/hedera-app/src/main/java/com/hedera/node/app/grpc/impl/netty/NettyGrpcServerManager.java

```diff
@@ -269,7 +269,7 @@ private NettyServerBuilder builderFor(final int port, NettyConfig config) {
                     .bossEventLoopGroup(new EpollEventLoopGroup())
                     .workerEventLoopGroup(new EpollEventLoopGroup());
             logger.info("Using Epoll for gRPC server");
-        } catch (final Throwable ignored) {
+        } catch (final Exception ignored) {
```

Comment:
This is not correct.  The primary error encountered here (at least the *expected* one) is UnsatisfiedLinkError, which is an Error, not an Exception.  We'll need to fix this in order for integration or e2e tests to run on Mac and Windows.

Reply:
Interesting, so the reason the PR checks did not fail is because we use Linux in CI, right? Have we considered adding Windows and Mac runners to CI so that we catch this kind of issues earlier? cc @netopyr